pkg/node/node.go

```diff
@@ -239,6 +227,38 @@ func NewBee(o Options) (*Bee, error) {
 		b.debugAPIServer = debugAPIServer
 	}
 
+	var wg sync.WaitGroup
+
+	// TODO: be more resilient on connection errors and connect in parallel
+	for _, a := range o.Bootnodes {
+		wg.Add(1)
+		go func(aa string) {
+			defer wg.Done()
+			addr, err := ma.NewMultiaddr(aa)
+			if err != nil {
+				logger.Debugf("multiaddress fail %s: %w", aa, err)
+				logger.Errorf("connect to bootnode %s: %w", aa, err)
```

Comment:
I think that Error message should not expose the internal error, and that it may be better just to have "failed to connect to bootnode %s".

This goes to other lines below.

Reply:
the error here is about multiaddress parse, why do we mention that we can't connect to the bootnode?

Reply:
@janos I agree, it is a mistake :)
@acud My reasoning was that this is a users message, so user does not care why  the connect failed. Does it make sense?